src/main/java/poomasi/domain/reservation/service/ReservationPlatformService.java

```diff
@@ -0,0 +1,69 @@
+package poomasi.domain.reservation.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import poomasi.domain.farm._schedule.entity.FarmSchedule;
+import poomasi.domain.farm._schedule.entity.ScheduleStatus;
+import poomasi.domain.farm._schedule.service.FarmScheduleService;
+import poomasi.domain.farm.entity.Farm;
+import poomasi.domain.farm.service.FarmService;
+import poomasi.domain.member.entity.Member;
+import poomasi.domain.member.service.MemberService;
+import poomasi.domain.reservation.dto.request.ReservationRequest;
+import poomasi.domain.reservation.dto.response.ReservationResponse;
+import poomasi.domain.reservation.entity.Reservation;
+import poomasi.global.error.BusinessError;
+import poomasi.global.error.BusinessException;
+
+@Service
+@RequiredArgsConstructor
+public class ReservationPlatformService {
+    private final ReservationService reservationService;
+    private final MemberService memberService;
+    private final FarmService farmService;
+    private final FarmScheduleService farmScheduleService;
+
+    private final int RESERVATION_CANCELLATION_PERIOD = 3;
+
+    public ReservationResponse createReservation(ReservationRequest request) {
+        Member member = memberService.findMemberById(request.memberId());
+        Farm farm = farmService.getValidFarmByFarmId(request.farmId());
+        FarmSchedule farmSchedule = farmScheduleService.getValidFarmScheduleByFarmIdAndDate(request.farmId(), request.reservationDate());
+
+        // TODO: 예약 가능한지 확인하는 로직 추가
+
+        Reservation reservation = reservationService.createReservation(request.toEntity(member, farm, farmSchedule));
+
+
+        return reservation.toResponse();
+    }
+
+    public ReservationResponse getReservation(Long memberId, Long reservationId) {
+        Reservation reservation = reservationService.getReservationById(reservationId);
+        if (!reservation.getMember().getId().equals(memberId) || !memberService.isAdmin(memberId)) {
+            throw new BusinessException(BusinessError.RESERVATION_NOT_ACCESSIBLE);
+        }
+
+        return reservation.toResponse();
+    }
+
+    public void cancelReservation(Long memberId, Long reservationId) {
+        Reservation reservation = reservationService.getReservationById(reservationId);
+
+        if (!reservation.getMember().getId().equals(memberId) || !memberService.isAdmin(memberId)) {
+            throw new BusinessException(BusinessError.RESERVATION_NOT_ACCESSIBLE);
+        }
```

Comment:
메서드 분리를 통해 로직을 좀 더 간결하게 표현해볼 수 있을 것 같아요.
그리고 memberService.isAdmin(memberId) 로직은 Resevation 의 Member 객체의 값을 활용하면 없앨 수 있을 것 같은데 어떻게 생각하시나요?

```suggestion
        if (reservation.isAvaliableCancel(memberId)) {
            throw new BusinessException(BusinessError.RESERVATION_NOT_ACCESSIBLE);
        }
```